File: InsuranceAdminPanelClient/src/app/entities/agreement.ts
import { Person } from "./person";

export class Agreement{
  public agreementName:string = "";
  public policyId:number = 0;
  public totalPrice:number = 0;
  public installment = 1;
  public person:Person = new Person();

  public startDate:Date = new Date();
  public endDate:Date = new Date();

  constructor(){
  }

}

File: Insurance/Insurance.Entity/Models/Agreement.cs
using Insurance.Entity.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Insurance.Entity.Models
{
    public class Agreement : EntityBase
    {
        public int AgreementId { get; set; }
        public string AgreementName { get; set; }
        public int PolicyId { get; set; }
        public double TotalPrice { get; set; }
        public int Installment { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public Person Person { get; set; }

    }
}
